source/_integrations/starline.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "StarLine"
+description: "Instructions on how to setup your StarLine account with Home Assistant."
+logo: starline.png
+ha_category:
+  - Car
+  - Binary Sensor
+  - Presence Detection
+  - Lock
+  - Sensor
+  - Switch
+ha_release: "0.100"
+ha_iot_class: Cloud Polling
+ha_qa_scale: gold
```

Comment:
@frenck can you check this?

Reply:
The `ha_qa_scale` is incorrect for this integration. For example, the gold level requires integration tests:

> Tests for reading data from/controlling the integration

Currently, only tests for the configuration flow are provided, hence, it does not meet the gold requirements.

Please adjust the `ha_qa_scale` to a matching level or remove it completely.

For a developers checklist, please see:

<https://developers.home-assistant.io/docs/en/integration_quality_scale_index.html>